infoenergia_api/api/reports/reports.py

```diff
@@ -19,7 +19,7 @@ class ReportsView(PaginationLinksMixin, HTTPMethodView):
 
     async def post(self, request, user):
         logger.info("Uploading contracts")
-        report_ids, month = await get_report_ids(request)
+        report_ids, month, report_type = await get_report_ids(request)
 
         bapi = await BeedataApiClient.create(
```

Comment:
I know, call reports endpoint is not so common now. But for each request we are creating a new fresh connection to Beedata api. Maybe we can encapsulate this in a manager class like in OV with the ERP connection.